ddl/index.go

```diff
@@ -605,6 +610,7 @@ func (w *worker) onCreateIndex(d *ddlCtx, t *meta.Meta, job *model.Job, isPK boo
 		}
 		job.SchemaState = model.StateDeleteOnly
 		metrics.GetBackfillProgressByLabel(metrics.LblAddIndex, job.SchemaName, tblInfo.Name.String()).Set(0)
+		metrics.GetBackfillProgressByLabel(metrics.LblAddIndexMerge, job.SchemaName, tblInfo.Name.String()).Set(0)
```

Comment:
We've differentiated the label in https://github.com/pingcap/tidb/pull/37709/files#diff-51926b25b437247805c6880b2fd041f096575b30050b560ab934f1d863bb6bf6R322, so is `LblAddIndex` enough here?